### ISSUE TITLE
Extension should be quiet in workspaces without jest

### ISSUE THREAD

**Issue by AArnott** [kavod-io/vscode-jest-test-adapter#85]
**Describe the bug**

When the Jest Test Explorer extension is installed, it tries to discover tests even in a workspace that doesn't use jest. As a result, I see an error in the Test Explorer:

![image](https://user-images.githubusercontent.com/3548/93492406-ffae1300-f8c7-11ea-852a-f2d1b7b48643.png)

This leads me to disable the Jest Test Explorer extension for these workspaces, which is a pain and doesn't support well when I've switched to jest in a workspace in some branches but other branches still use jasmine.

Can the extension look just once for `jest` to be installed (perhaps by checking package.json or under node_modules) and quietly no-op if it isn't there? 
Tip: The "Jest" extension itself seems to key off of the presence of a `jest.config.js` file in the workspace. Maybe the extension can do the same thing.

**Repro steps**

After VS Code has the Jest Test Explorer extension installed:

```
yarn init -y
code .
```

Navigate to Test Explorer to see it's already showing an error.

**Desktop (please complete the following information):**
 - OS: Windows

**Comment by AArnott** [comment]
Ah, I notice that only the top-left error comes from the Jest Test Explorer. The bottom two come from the Jest extension. I'll file another issue to track those.

**Comment by rossknudsen** [comment]
I think the offending code is this:

```ts
// StandardParser.ts

const isStandard = async (workspaceRoot: string): Promise<boolean> => {
  const packageJsonPath = path.resolve(workspaceRoot, "package.json");
  // TODO we should also check that the jest package is installed, or that we are using a globally installed one.
  // Otherwise, this may not be a Jest project.
  return await exists(packageJsonPath);
};
```

So I probably should ensure that `jest` exists in the package.json file (as per my TODO 😉).  I also need to handle the case where users have installed globally and hence it won't be in their package file.  But that is probably a separate issue.

**Comment by rossknudsen** [comment]
This is hopefully fixed by the latest release (0.7.3).  I'll close this issue, but if you find that the issue persists, please reopen.

**Comment by AArnott** [comment]
Thanks.
